apps/challenges/migrations/0086_challenge_requirements.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.20 on 2021-08-11 06:51
```

Comment:
@vinceli1004 we should remove this migration file as we are not making any model changes